README.md

```diff
@@ -122,6 +122,7 @@ If you cannot find suitable test data on this repository, please contact us on t
       * 'fast5.tar.gz': compressed `fast5` folder with the following structure: `fast5/100read_fast5/*.fast5`
     * fastq
       * 'test.fastq.gz'
+      * 'test_2.fastq.gz': Contains more data than 'test.fastq.gz', which is needed to test MetaMaps
```

Comment:
```suggestion
      * 'test_2.fastq.gz'
```

I think it's fine to just have a second test dataset. It would be good to give a accession code if you've got this from public data somewhere (and specify if ou've subsampled)

Reply:
Unfortunately this data is not public and won't be made public either (beyond being public on github). We just generated it for validation of the workflow.